themes/src/main/resources/theme/keycloak.v2/login/resources/css/styles.css

```diff
@@ -1,5 +1,5 @@
 :root {
-    --keycloak-logo-url: url('../img/keycloak-logo-text.png');
```

Comment:
This original image can be removed, it doesn't seem to be referenced anywhere else.

Reply:
Yes, I'll do it. I was only wondering whether some custom users' themes reference the logo to not provide any breaking change, but I think it should be ok.

Reply:
I considered that as well, but we don't really provide those kinds of guarantees for themes historically as far as I can tell.